### ISSUE TITLE
Edit File and Notes of uploaded files

### ISSUE THREAD

**Issue by roratonet** [grokability/snipe-it#8720]
**Is your feature request related to a problem? Please describe.**
After uploading a file to an **Asset** or **License**, I'm not able to edit the **Notes** or replace files afterwards

**Describe the solution you'd like**
Edit the notes and replace files of uploaded entries

**Comment by snipe** [comment]
> Edit the notes and replace files of uploaded entries

Why would you replace the file (versus deleting and uploading a new one)

**Comment by roratonet** [comment]
> > Edit the notes and replace files of uploaded entries
> 
> Why would you replace the file (versus deleting and uploading a new one)

I have no issues in deleting a previously uploaded file and uploading a new one.
But I find it more useful to _edit_ the **Notes** in case of need, without having to
1) download the file
2) remove the entry
3) create a new entry with different **Note** and re-upload the file previously downloaded file.

With this behaviour I would alter the **Created at** timestamp, which in my opinion is bad. Maybe an additional **Modified at** column could be useful then.

**Comment by snipe** [comment]
I don't think we plan on implementing this - the whole point of the `action_log` is that it's an immutable record.
